packages/primitives/src/index.js

```diff
@@ -8,6 +8,12 @@ export const Note = 'NOTE';
 export const Path = 'PATH';
 export const Rect = 'RECT';
 export const Line = 'LINE';
+export const Form = 'FORM';
+export const FormField = 'FORM_FIELD';
+export const FormText = 'FORM_TEXT';
+export const FormPushButton = 'FORM_PUSH_BUTTON';
```

Comment:
What do you think about renaming this to `Button` to match react native (web) primitives? Not sure either what purpose this element has